examples/job_queue/job_docker.yaml

```diff
@@ -18,7 +18,7 @@ setup: |
 run: |
   timestamp=$(date +%s)
   conda env list
-  for i in {1..180}; do
+  for i in {1..300}; do
```

Comment:
Do we need to increase the time here? Does this mean the newly added commands introduce significant overheads?

Reply:
IIRC I think it is just Azure being too slow... Originally we does not test on Azure. Lemme test the original time w/ AWS/GCP again

Reply:
Yes it is only for Azure. Now only setting to 300 for Azure